scripts/test-w-coverage.sh

```diff
@@ -19,7 +19,7 @@ go test -timeout 120s \
    -covermode count "${modules[@]}" | tee testoutput.txt || { echo "go test returned non-zero"; exit 1; }
 
 # shellcheck disable=SC2002
-cat coverage.txt.with_generated_code | grep -v "_generated.go" > coverage.txt
+cat coverage.txt.with_generated_code | grep -v "_generated.go" | grep -v "fake.go" > coverage.txt
```

Comment:
Maybe wildcard so someone doesn't end up creating `fake_foo.go`. Would be tough to argue to name all fakes as `fake.go`, and enforce it in review.